dubbo-config/dubbo-config-api/src/main/java/org/apache/dubbo/config/ServiceConfig.java

```diff
@@ -205,6 +205,8 @@ public synchronized void export() {
             return;
         }
 
+        delay = getGlobalServiceDelay();

```

Comment:
Your pr may have a problem.
If you directly override the delay here, the delay attribute in the XML file will never take effect.